app/views/docs/view.html.erb

```diff
@@ -1,2 +1,49 @@
-
-<%= raw @result %>
+<section class="cli-header bg-light px-3 py-3 pt-md-5 pb-md-4 mx-auto border-bottom">
+  <div class="container cointaier-shrinked">
+    <h1 class="display-4 text-left text-success">
+      Docs
+      <button class="navbar-toggler float-right mt-4 d-md-none" type="button" data-toggle="collapse" data-target="#docs-sidebar" aria-controls="docs-sidebar" aria-expanded="false" aria-label="Toggle Docs Navigation">
+        <span><i class="fa fa-bars"></i></span>
+      </button>      
+    </h1>
+  </div>
+</section>
+<section class="pt-8 pt-md-11 bg-dark p-3 p-md-5 border-top">
+  <div class="container text-white">
+    <div class="row">
+        <div class="col-lg-3 col-md-3 col-sm-12 d-sm-flex bg-light pt-2 pl-0 collapse d-md-show navbar-collapse" id="docs-sidebar">
+          <div class="card bg-secondary">
+            <div class="card-body text-white">
+              <ul class="nav flex-column flex-nowrap overflow-hidden">                
+                <% directory_hash("#{Rails.root}/public/documentation")[:children].reverse_each do |item| %>
+                  <li class="nav-item">
+                    <% if item[:children].present? %>
+                      <a class="nav-link collapsed text-truncate text-white" href="#doc-<%= item[:text].downcase %>" data-toggle="collapse" data-target="#doc-<%= item[:text].downcase %>">
+                        <span>&#8226; <%= item[:text].titleize  %></span>
+                      </a>
+                      <div class="collapse <%= @section.downcase == item[:text].downcase ? 'show' : '' %>" id="doc-<%= item[:text].downcase  %>" aria-expanded="false">
```

Comment:
When you are in a given category, let's say you are in Cloud Hosting > Logs, it should open Cloud Hosting when the page loads

Reply:
seems working well, except for the scheduler page, maybe because of the spaces?

![image](https://user-images.githubusercontent.com/8007722/71911205-4d6f5a80-3141-11ea-949c-2c214bebf0d1.png)